src/spikeinterface/postprocessing/template_metrics.py

```diff
@@ -141,8 +145,29 @@ def _set_params(
             metrics_kwargs_ = _default_function_kwargs.copy()
             metrics_kwargs_.update(metrics_kwargs)
 
+        all_metric_names = metric_names
```

Comment:
I got a bit confused with `all_metric_names`, `metric_names` and `existing_metric_names`. Could these be renamed to be slightly more informative? I am quite naive to the `template_metrics` / `sorting_analyzer` stuff but am a little confused by what is going on here, could a short comment be added at the top of this section to explain?

Reply:
I think (if my below understanding is correct) then maybe just `all_metric_names` -> `metric_names_to_compute` could work? (+ the short comment explainer)